rpcserver.go

```diff
@@ -1262,18 +1296,33 @@ func (r *rpcServer) GetInfo(ctx context.Context,
 		NumPendingChannels:  nPendingChannels,
 		NumActiveChannels:   activeChannels,
 		NumPeers:            uint32(len(serverPeers)),
-		BlockHeight:         uint32(bestHeight),
-		BlockHash:           bestHash.String(),
-		SyncedToChain:       isSynced,
-		Testnet:             isTestnet(&activeNetParams),
-		Chains:              activeChains,
+		Chains:              chains,
 		Uris:                uris,
 		Alias:               nodeAnn.Alias.String(),
-		BestHeaderTimestamp: int64(bestHeaderTimestamp),
 		Version:             version(),
 	}, nil
 }
 
+func FindPeerChain (chains *[]*lnrpc.PeerChain, chain chainCode ) (*lnrpc.PeerChain){
```

Comment:
This function could use a comment to explain what it does. It looks like it's attempting to find a chain by name in a list of chains, and if none is found it creates one with just the name and adds it to the list?